phoenix-ios/phoenix-ios/views/configuration/general/liquidity management/LiquidityAdsView.swift

```diff
@@ -0,0 +1,1006 @@
+)
+#else
+fileprivate var log = Logger(OSLog.disabled)
+#endif
+
+struct LiquidityAdsView: View {
+	
+	let liquidityOptions: [Bitcoin_kmpSatoshi] = [
+		Bitcoin_kmpSatoshi(sat:    100_000),
+		Bitcoin_kmpSatoshi(sat:    250_000),
+		Bitcoin_kmpSatoshi(sat:    500_000),
+		Bitcoin_kmpSatoshi(sat:  1_000_000),
+		Bitcoin_kmpSatoshi(sat:  2_000_000),
+		Bitcoin_kmpSatoshi(sat: 10_000_000)
+	]
+		
+	@State var channels: [LocalChannelInfo] = []
+	@State var balanceMsat: Int64? = nil
+	@State var mempoolRecommendedResponse: MempoolRecommendedResponse? = nil
+	
+	@State var sliderValue: Double = 0
+	@State var feeInfo: LiquidityFeeInfo? = nil
+	@State var finalResult: Lightning_kmpChannelCommand.CommitmentSpliceResponse? = nil
+	
+	@State var isEstimating: Bool = false
+	@State var isPurchasing: Bool = false
+	@State var isPurchased: Bool = false
+	
+	@State var popoverPresent_minerFee = false
+	@State var popoverPresent_serviceFee = false
+	@State var popoverPresent_duration = false
+	
+	@EnvironmentObject var currencyPrefs: CurrencyPrefs
+	
+	var popoverPresent: Bool {
+		return popoverPresent_minerFee || popoverPresent_serviceFee || popoverPresent_duration
+	}
+	
+	// --------------------------------------------------
+	// MARK: View Builders
+	// --------------------------------------------------
+	
+	@ViewBuilder
+	var body: some View {
+		
+		content()
+			.navigationTitle("Add Liquidity")
+			.navigationBarTitleDisplayMode(.inline)
+	}
+	
+	@ViewBuilder
+	func content() -> some View {
+		
+		List {
+			section_info()
+			if let finalResult {
+				section_result(finalResult)
+			} else {
+				section_settings()
+				if let feeInfo {
+					section_estimatedCost(feeInfo)
+				} else {
+					section_estimateCostButton()
+				}
+			}
+		}
+		.listStyle(.insetGrouped)
+		.listBackgroundColor(.primaryBackground)
+		.onReceive(Biz.business.peerManager.channelsPublisher()) {
+			channelsChanged($0)
+		}
+		.onReceive(Biz.business.balanceManager.balancePublisher()) {
+			balanceChanged($0)
+		}
+		.task {
+			await fetchMempoolRecommendedFees()
+		}
+	}
+	
+	@ViewBuilder
+	func section_info() -> some View {
+		
+		Section {
+			
+			VStack(alignment: HorizontalAlignment.leading, spacing: 24) {
+				
+				Text("**Imagine that your wallet is a bucket**, and your balance is the water in the bucket.")
+				
+				HStack(alignment: VerticalAlignment.top, spacing: 8) {
+					Image("bucket")
+						.resizable()
+						.scaledToFit()
+						.frame(maxWidth: 50, maxHeight: 50)
+					
+					Text(
+						"""
+						Spending = pouring water out
+						Receiving = adding more water
+						"""
+					)
+				} // </HStack>
+				
+				Text(
+				"""
+				If the bucket needs to be resized to allow for more water, \
+				that will require an on-chain operation with a mining fee.
+				""")
+				
+				Text(
+				"""
+				You may be able to reduce your fees by increasing your bucket size \
+				ahead of a series of incoming payments.
+				""")
+				
```

Comment:
There are several issues with this message:
- quite long (it's scary and people won't read it)
- takes a lot of screen space (on my phone I have to scroll down to reach the interesting part)
- the bucket analogy is kinda advanced and also already in the [FAQ](https://phoenix.acinq.co/faq#what-is-inbound-liquidity).

So the message should be as short as possible (something around "planning ahead to save money later"), with a link to the FAQ for users who want to know more. The FAQ is easier to maintain (shared between iOS and Android), and can be easily updated if we want to add more information.